vignettes/big_ns.Rmd

```diff
@@ -0,0 +1,176 @@
+    "g1", "rowlabel1",  "column cols", "col 4"   , "col4"  ,  "value",    1,
+    "g1", "rowlabel1",             NA,         NA, "mycol5",  "value",    1,
+    "g1", "rowlabel2",  "column cols", "cols 1,2", "col1"  ,  "value",    2,
+    "g1", "rowlabel2",  "column cols", "cols 1,2", "col2"  ,  "value",    2,
+    "g1", "rowlabel2",             NA,        NA , "mycol3",  "value",    2,
+    "g1", "rowlabel2",  "column cols", "col 4"   , "col4"  ,  "value",    2,
+    "g1", "rowlabel2",             NA,         NA, "mycol5",  "value",    2,
+    "g2", "rowlabel3",  "column cols", "cols 1,2", "col1"  ,  "value",    3,
+    "g2", "rowlabel3",  "column cols", "cols 1,2", "col2"  ,  "value",    3,
+    "g2", "rowlabel3",             NA,         NA, "mycol3",  "value",    3,
+    "g2", "rowlabel3",  "column cols", "col 4"   , "col4"  ,  "value",    3,
+    "g2", "rowlabel3",             NA,         NA, "mycol5",  "value",    3,
+  )
+
+
+
+spanning_tfrmt<- tfrmt(
+    group = group,
+    label = label,
+    param = parm,
+    values = val,
+    column = c(span2, span1, my_col),
+    body_plan = body_plan(
+      frmt_structure(group_val = ".default", label_val = ".default", frmt("x"))
+    ),
+    col_plan = col_plan(
+      group,
+      label,
+      starts_with("col")
+    )
+  )
+
+print_to_gt(spanning_tfrmt, data)
+```
+ 
+ If we just wanted to put the big N on the highest "column cols" we would need to add a row to the ARD where `span2` equals `"column cols"` and all other column columns are missing. 
+```{r}
+with_big_n_data <- tribble(
+  ~group,     ~label,        ~span2,  ~span1,     ~my_col,    ~parm,   ~val,
+      NA,        NA,  "column cols",     NA,       NA  ,  "bigN",    18,
+) %>% 
+  bind_rows(data)
+
+#Now we can add the big_n to the tfrmt from before 
+n_span_tfrmt <- spanning_tfrmt %>% 
+  tfrmt(big_n = big_n_structure(param = "bigN", n_frmt = frmt(" N = xx"))) 
+
+print_to_gt(n_span_tfrmt, .data = with_big_n_data)
+```
+ 
+ If we also wanted big N's on `col 1,2`, `col4`, and `mycol3`. So we will need Add 3 more rows to our ARD. One that goes down to teh `span1` level of specificity and two that go all the way down to `my_col` level of specificity. (Note: because we are keeping the `parm` values the same we can use the same tfrmt)
+```{r}
+with_more_big_n_data <- tribble(
+  ~group,     ~label,        ~span2,  ~span1,     ~my_col,    ~parm,   ~val,
+    NA,         NA,  "column cols",  "cols 1,2",   NA  ,      "bigN",    12,
+    NA,         NA,  "column cols",  "col 4"   ,   "col4"  ,  "bigN",    6,
+    NA,         NA,             NA,          NA,   "mycol3",  "bigN",    6
+) %>% 
+  bind_rows(with_big_n_data)
+
+print_to_gt(n_span_tfrmt, .data = with_more_big_n_data)
+```
+ 
+As a final note, if column names get changed in the `col_plan`, it shouldn't effect the ARD. The column names for the big N's should match the ARD column names not any future names. But, if names do get updated the big N's will be uneffected. 
```

Comment:
this reads a little confusing to me as written. Did you mean that the col_plan does not change the ARD?

Reply:
Well yes and no.... basic if you rename a column in a col plan you shouldn't have a your big N's like 
```{r}
tribble(
~column, ~param, ~val,
"New Name", "big N", 10
)
```